lib/jekyll-import/importers/wordpressdotcom.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+require 'uri'
```

Comment:
Importers load dependencies via the `self.require_deps` method.
We've already listed `open-uri` to be loaded at runtime. Would that suffice..?

Reply:
What I meant was, you need not explicitly `require 'uri'` since we're already loading `open-uri`.
I just checked to confirm.
You can call `URI()` without explicitly `require-ing 'uri'`